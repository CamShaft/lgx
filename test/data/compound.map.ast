#{
  type => map,
  line => 1,
  children => #{
    first_key => #{
      type => literal,
      value => first,
      line => 2
    },
    second_key => #{
      type => literal,
      value => second,
      line => 3
    },
    third_key => #{
      type => literal,
      value => third,
      line => 4
    }
  }
}.