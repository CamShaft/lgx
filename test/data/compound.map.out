#{
  first_key => first,
  second_key => second,
  third_key => third
}.